packages/pages/src/dev/server/middleware/serverRenderSlugRoute.ts

```diff
@@ -17,22 +17,28 @@ type Props = {
   vite: ViteDevServer;
   dynamicGenerateData: boolean;
   projectStructure: ProjectStructure;
+  defaultLocale: string;
 };
 
 export const serverRenderSlugRoute =
-  ({ vite, dynamicGenerateData, projectStructure }: Props): RequestHandler =>
+  ({
+    vite,
+    dynamicGenerateData,
+    projectStructure,
+    defaultLocale,
+  }: Props): RequestHandler =>
   async (req, res, next): Promise<void> => {
     try {
       const url = new URL("http://" + req.headers.host + req.originalUrl);
-      const locale = req.query.locale?.toString() ?? "en";
+      const locale = req.query.locale?.toString() ?? defaultLocale;
       const slug = decodeURI(url.pathname.substring(1));
 
       const templateFilepaths =
         getTemplateFilepathsFromProjectStructure(projectStructure);
       const matchingStaticTemplate: TemplateModuleInternal<any, any> | null =
         await findMatchingStaticTemplate(vite, slug, templateFilepaths);
       if (matchingStaticTemplate) {
-        sendStaticPage(
+        await sendStaticPage(
```

Comment:
`sendStaticPage` returns a promise. Without this await, if it doesn't match a locale, the server crashes. This await keeps the server up and sends the standard error 500 page.

I presume this is what we want, but I'm not sure if there's a reason `sendStaticPage` wasn't awaited previously

Reply:
Most likely a bug